src/test/java/co/refiere/resources/CampaignResourceTest.java

```diff
@@ -89,6 +89,12 @@ public void createCampaignTest(){
         testCampaign = campaignDao.getCampaigById(object.getInt("campaignId"));
     }
     
+    @Test
+    public void getCampaigns(){
+        final Response response = target().path("v1/campaign/company/1/campaignList").request().get();
+        Assert.assertNotNull(response);
```

Comment:
Also check if the returned data is consistent ( entered campaign for testing is returned ).